misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -0,0 +1,150 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import os
+import re
+import subprocess
+import sys
+from collections import OrderedDict
+from typing import Dict, List
+
+import junit_xml
+
+from materialize import ROOT, ci_util
+
+Coverage = Dict[str, OrderedDict[int, int]]
+SOURCE_RE = re.compile(
+    "^/var/lib/buildkite-agent/builds/buildkite-.*/materialize/coverage/(.*$)"
+)
+
+
+def find_modified_lines() -> Coverage:
+    """
+    Find each line that has been added or modified in the current pull request.
+    """
+    base_branch = os.getenv("BUILDKITE_PULL_REQUEST_BASE_BRANCH", "main") or os.getenv(
```

Comment:
I think the `os.getenv(
        "BUILDKITE_PIPELINE_DEFAULT_BRANCH", "main"
    )
`

part will not run, as the first os.getenv is guaranteed to not return `None`

Reply:
I thought so too initially, then actually ran into the problem that on this PR I managed to break BUILDKITE_PIPELINE_DEFAULT_BRANCH so that it is set to an empty string, but still set.